channeldb/payments.go

```diff
@@ -116,6 +117,50 @@ func (db *DB) FetchAllPayments() ([]*OutgoingPayment, error) {
 	return payments, nil
 }
 
+// LookupPayment attempts to look up an payment according to its 32 byte
+// payment hash.
+func (db *DB) LookupPayment(paymentHash [32]byte) (*OutgoingPayment, error) {
+	var payment *OutgoingPayment
+
+	err := db.View(func(tx *bolt.Tx) error {
+		payments := tx.Bucket(paymentBucket)
+		if payments == nil {
+			return ErrNoPaymentsCreated
+		}
+
+		return payments.ForEach(func(k, v []byte) error {
```

Comment:
Try to avoid single letter variable names. It wasn't clear to me what "k" and "v" were until I read the comment below that mentions a "value". Maybe "rawPayment" would be a good name in place of "v".

Also it looks like k is unused, so I think you can replace it with "_". As is, I scanned the whole function to figure out where it was used, only to realize after a minute that it wasn't.